src/components/player/PlayerBar.tsx

```diff
@@ -81,7 +86,7 @@ export const PlayerBar = React.memo(
     }, [location]);
 
     function toggleFullPlayer() {
-      setFullPlayer((prev) => !prev);
+      setFullPlayer(!fullPlayer);
```

Comment:
Should the value assigned to `setFullPlayer` be a function type like before changed? (Sorry, not familiar with these differences, but I commented that I edited this and it could be a possible cause of the bug.)